src/core/load-fragment.c

```diff
@@ -1711,7 +1713,15 @@ int config_parse_exec_cpu_affinity(
                 return 0;
         }
 
-        r = parse_cpu_set_extend(rvalue, &c->cpu_set, true, unit, filename, line, lvalue);
+        r = unit_full_printf(u, rvalue, &k);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "Failed to resolve unit specifiers in '%s'",
+                           rvalue);
+                return -ENOEXEC;
```

Comment:
I do not think this should be critical. Please downgrade the log level to `LOG_WARNING`, add `", ignoring"` in the log message, and `return 0;`:
```
log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to resolve unit specifiers in '%s', ignoring: %m");
return 0;
```

Reply:
Ready, thank you